applications/api_test.py

```diff
@@ -138,6 +138,24 @@ def test_get_or_create_bootcamp_application(mocker):
     assert created is False
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize("has_resume,has_linkedin,expected", [
+    (True, False, AppStates.AWAITING_USER_SUBMISSIONS.value),
+    (True, True, AppStates.AWAITING_USER_SUBMISSIONS.value)
```

Comment:
You don't need the `AppStates.AWAITING_USER_SUBMISSIONS.value` in the test parameters since you're expecting the same value in all cases